modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc
+
+:_content-type: PROCEDURE
+[id="machine-user-infra-machines-ibm-z_{context}"]
+= Creating {op-system} machines on {ibmzProductName} with z/VM
+
+You can create more {op-system-first} compute machines running on {ibmzProductName} with z/VM and attach them to your existing bare metal x86_64 cluster.
```

Comment:
attach them to your existing bare metal x86_64 cluster. -> attach them to your existing cluster.